### DISCUSSION TITLE
Ip Address

### DISCUSSION THREAD

**Discussion by LeMaskerade** [IceWhaleTech/CasaOS#1686] [Q&A]
i had to restart my server.
how do i find out what the ip address is which casa os is running on?

**Comment by TheOneAndOnlySpecial** [comment]
Hi there, 
Assuming that you have access to the terminal of the machine, you can run the command hostname -I. This showed the local ip and allowed me to find my server again. This worked for me, but if it doesn't work, then you can google "how to find local ip in linux" and that should be able to get you there.
